### PR TITLE
TMEDIA-767 : Alias Tokens

### PR DESCRIPTION
**Pull request by ecarlisle**
## Ticket

- [TMEDIA-767](https://arcpublishing.atlassian.net/browse/TMEDIA-767)

## Description

1. This PR introduces alias tokens provided by design. 
2. Any other changes are due to using the [token configuration](https://github.com/WPMedia/arc-themes-feature-pack/blob/commerce/site-styles/arc-commerce/_index.scss#L2-L550) from the [arc-themes-feature-pack](https://github.com/WPMedia/arc-themes-feature-pack/) repository (the source of truth).
3. Two existing alias tokens were renamed:
    a. `primary-font-family` was renamed to `font-family-primary`.
    b. `primary-color` was changed to `color-primary`.

## Acceptance Criteria

Update alias Scss map that contains:
1. Alias key names (Friendly naming) - See Figma resource for alias tokens currently used by design
2. Names should be split by a hyphen
5. Initial values that reference global tokens where possible (Font Family is likely to be an exception to the rule here)

## Test Steps

1. Checkout branch - `git checkout TMEDIA-767`
2. Update dependencies - `npm i`
3. Run Storybook `npm run storybook`
4. Sass should build without error.
5. Components and blocks should be using the `font-family-primary` and `primary-color` tokens as expected.

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [ ] Confirmed all the test steps a reviewer will follow above are working.
- [ ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ ] Confirmed relevant documentation has been updated/added.
- [ ] Add label - **ready for review** when the pull request is ready for someone to begin reviewing

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] All GitHub Actions pass
- [ ] Ran the code locally based on the test instructions.
- [ ] Checked Chromatic for Storybook changes, accepted the updates if acceptable
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.
- [ ] Approve and Add label - **ready to merge** if you are happy with the pull request
- [ ] Want another reviewer? Add the label **additional review**

### CODE REVIEW

File: .storybook/themes/commerce.scss [hunk 1]
```diff
@@ -3,50 +3,619 @@
...
+		"z-index-1": 1,
+		"z-index-2": 10,
+		"z-index-3": 50,
+		"z-index-4": 100,
+		"z-index-5": 500,
+		"border-radius-1": 0.25em,
+	),
+	$alias: (
+		// Font families
+		"font-family-primary": "Inter",
```
Comment (brentmiller1973) [inline .storybook/themes/commerce.scss:161]: No default sans and serif specified?  Should the base fonts be specified as global maybe

Comment (brentmiller1973) [reply .storybook/themes/commerce.scss:161]: Ex: , "font-family-primary": "Inter", var(--serifFallBackFont);

**Review by brentmiller1973 — CHANGES_REQUESTED** [review]
Think we need to consider fallback fonts for serif and non-serif

**Comment by ecarlisle** [comment]
Outdated PR with changed scope. Closing.

**Pull request closed without merging**